Tests/CommandsTests/TestToolTests.swift

```diff
@@ -55,4 +55,26 @@ final class TestToolTests: XCTestCase {
         }
       #endif
     }
+    
+    func testNumWorkersParallelRequeriment() throws {
+        fixture(name: "Miscellaneous/EchoExecutable") { path in
```

Comment:
Because of reasons, these tests won't work on Linux CI. Can you wrap them similar to this: https://github.com/apple/swift-package-manager/blob/master/Tests/FunctionalTests/MiscellaneousTests.swift#L213

Reply:
Also run `.build/x86_64-apple-macosx10.10/debug/spm test --generate-linuxmain` to update the linux main file.

Reply:
Interesting. Done 👍